src/components/footer.tsx

```diff
@@ -0,0 +1,66 @@
+import cn from 'classnames';
+
+type Props = {
+  selectedFilter: string;
+  onSelect: (a: string) => void;
+  count: number;
+};
+
+export const Footer: React.FC<Props> = ({
+  selectedFilter,
+  onSelect,
+  count,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${count} items left`}
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: selectedFilter === 'All',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => onSelect('All')}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={cn('filter__link', {
+            selected: selectedFilter === 'Active',
```

Comment:
pls create enum with values: 'All', 'Active' and 'Completed' and use where needed